Lib/plistlib.py

```diff
@@ -245,7 +245,11 @@ def end_false(self):
         self.add_object(False)
 
     def end_integer(self):
-        self.add_object(int(self.get_data()))
+        raw = self.get_data()
+        if raw.startswith('0x') or raw.startswith('0X'):
```

Comment:
`raw.startswith(('0x', '0X'))`
or
`raw[:2] in ('0x', '0X')`

You can also use `int(raw, 0)` to parse decimal and hexadecimal literals (also octal and binary).